### ISSUE TITLE
There should be a way to unlock locked starting scenarios globally through option menu for a game install

### ISSUE THREAD

**Issue by NegInfinity** [CleverRaven/Cataclysm-DDA#63065]
### Is your feature request related to a problem? Please describe.

At the moment I have 3 worlds in the installation folder and probably nuked at least five if not a dozen

I've seen labs before in other builds, with other characters, so having locked out starting scenarios adds nothing except irritation. 

What's more, in the latest build when I finally triggered the "what are they hiding" achievement (which failed to trigger at least once), that did not unlock starting scenarios and they're still unavailable in all worlds.

### Solution you would like.

Add a clear english text menu somewhere within "Options" menu which says "Unlock all starting scenarios", which would unlock the scenario for all existing and future worlds for this particular cdda install.

### Describe alternatives you have considered.

I suppose scrapping the whole idea is an option? While achievements are fun, scenario lockout does not appear to mesh with existing gameplay process or contribute to the experience in a meaningful way.

### Additional context

At the moment some conversation on reddit exists somewhere which speaks about "disabling meta progression", except supposedly it is tied to the world, character tombstone, and you have to sacrifice a character to unlock it? Neither terminology nor the process are clear. For example, it is not obvious what "Meta progression" means in the first place. And in addition the process does not function the same way a normal game would. Normally achievements are global, while local achievements can still be used in some games (for example, Factorio), they're usually tied to the install or player profile and are shared across worlds. Meaning the moment anyone reaches the labs, the expected behavior is for the starting scenarios to be unlocked forever for all future worlds and all characters. Because worlds are disposable.

**Comment by GuardianDll** [comment]
There is a button in the debug menu, that do exactly this

**Comment by NegInfinity** [comment]
> There is a button in the debug menu, that do exactly this

There's no functionality in the *main* menu that would do what I describe, meaning unlock the starting scenario for all existing worlds or for all worlds.

There is "unlock all progression" menu in debug, but from description alone it is unclear what that would do.

It is also done in incredibly roundabout way, as it kills your current character.
![image](https://user-images.githubusercontent.com/15124172/211682377-9f389ba3-812d-4ea0-b449-47b2331d840d.png)
MEaning it requires a world, and a sacrifice and a debug menu access. 

Additionally good chunk of the warning it produces is over the top. 

The warning produced in "Meta progression" menu is similarly unhelpful.
![image](https://user-images.githubusercontent.com/15124172/211682685-b51412cb-4fa4-4a61-8cb8-3f6b7e44ef37.png)

In both case they are worded as if they're implied that unlocking the functioanlity would do some sort of drastic and irreversible changes to the world, like, for example setting all factions to hostile. It /sounds/ this way.

While in reality the only thing they should be doing would be unlocking lab start access.
Likewise unlocking definitely shouldn't require a sacrifice in order to work. As this is simply too many clicks for something basic.

**Comment by PatrikLundell** [comment]
Your suggestion is likely to be rejected. The powers that be decided upon this "solution" after the objections raised here had been voiced, and thus are unlikely to change their minds.

**Comment by Zireael07** [comment]
'Some scenarios are frustrating to new players" is true. I agree that the warning is a bit wordy but I fail t see where you read "drastic" changes into it.

Also: this option does more than just unlock lab starts...

**Comment by NegInfinity** [comment]
> 'Some scenarios are frustrating to new players" is true. I agree that the warning is a bit wordy but I fail t see where you read "drastic" changes into it.
> 
> Also: this option does more than just unlock lab starts...

And proposed change is adding a method to unlock the blocked starts. And not do whatever else this option is doing.

**Comment by RenechCDDA** [comment]
>In both case they are worded as if they're implied that unlocking the functioanlity would do some sort of drastic and irreversible changes to the world, like, for example setting all factions to hostile. It /sounds/ this way.

The warnings are completely accurate and true - disabling meta progression will instantly throw the covers off much of the lore and introduce you to concepts, situations, and factions that you will not be aware of until you are well versed in the game. It will also allow you to select many extremely challenging starting scenarios that will nearly guarantee your death even for experienced players. It does not imply in any way that it will 'set factions to hostile' or anything of the sort.

>Likewise unlocking definitely shouldn't require a sacrifice in order to work. As this is simply too many clicks for something basic.

All it takes is pressing "play now" followed by Q>Y. As previously mentioned, you can use the debug option if you prefer it. That is why it exists.
